From f5804a7c070ac01af04daf2dcdfb52186338b8bc Mon Sep 17 00:00:00 2001
From: Naiwen Yu <naiwyu@qti.qualcomm.com>
Date: Mon, 15 Sep 2025 15:35:10 +0800
Subject: [PATCH] limit systemd-coredump burst interval

In case of systemd-coredump itself abnormal, systemd-coredump service will keep being invoking,
finally cause thousands of coredump files of same issue in the system. Limit the StartLimitBurst
to avoid that.

Upstream-Status:Inappropriate
Change-Id:f5804a7c070ac01af04daf2dcdfb52186338b8bc
Signed-off-by: Naiwen Yu <naiwyu@qti.qualcomm.com>
---
 units/systemd-coredump@.service.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/units/systemd-coredump@.service.in b/units/systemd-coredump@.service.in
index 15bfb243b4..ab808d4915 100644
--- a/units/systemd-coredump@.service.in
+++ b/units/systemd-coredump@.service.in
@@ -42,3 +42,5 @@ StateDirectory=systemd/coredump
 SystemCallArchitectures=native
 SystemCallErrorNumber=EPERM
 SystemCallFilter=@system-service @mount
+StartLimitBurst=3
+StartLimitIntervalSec=60
-- 
2.34.1

